eng/mgmt/automation/generate_data.py

```diff
@@ -23,35 +23,9 @@
 YAML_BLOCK_REGEX = r'```\s?(?:yaml|YAML).*?\n(.*?)```'
 
 
-def sdk_automation(config: dict) -> List[dict]:
-    # priority:
-    # 1. autorestConfig from input
-    # 2. swagger/README.md in sdk repository that matches readme from input
-
+def get_sdk_readme_abspath(config: dict, readme_file_path: str) -> Optional[str]:
```

Comment:
this is actually `get_or_update_sdk_readme`?

Reply:
you are right, the func may get the readme file or update it, fixed